e2e/config.js

```diff
@@ -161,7 +161,8 @@ module.exports = {
     manageDocuments: 'Manage documents',
     uploadAdditionalApplications: 'Upload additional applications',
     manageOrders: 'Manage orders',
-    uploadWelshTranslations: 'Upload Welsh translations',
+    uploadWelshTranslations: 'Upload translated documents',
+    languageRequirement: 'Welsh language requirements',
```

Comment:
Is this stuff not from a separate pr?

Reply:
It's been merged, so I adapted the flag to disable the option in case is not true